learn/getting_started/cloud_quick_start.mdx

```diff
@@ -0,0 +1,81 @@
+
+Creating your project might take a few minutes. Once it is ready, you will see it in the project list. Click on the project name to go to the project overview page:
+
+![Meilisearch Cloud's main list of projects. It features only one project, "meilisearch-quick-start", and shows information such as API keys, URL, and number of monthly searches](https://raw.githubusercontent.com/meilisearch/documentation/cloud-quick-start/assets/images/cloud-getting-started/4-project-list.png)
+
+## Creating an index and adding documents
+
+After creating your project, you must index the data you want to search. Meilisearch stores and processes data you add to it in indexes. A single project may contain multiple indexes.
+
+First, click on the indexes tab in the project page menu:
+
+![The project overview page, featuring a secondary menu with several links. A red arrow points at a menu item: "Indexes"](https://raw.githubusercontent.com/meilisearch/documentation/cloud-quick-start/assets/images/cloud-getting-started/5-project-page-menu-indexes.png)
+
+This leads you to the index listing. Click on "Create an index":
+
+![An empty list of indexes in this project with a button on the upper right corner](https://raw.githubusercontent.com/meilisearch/documentation/cloud-quick-start/assets/images/cloud-getting-started/6-index-list-empty.png)
+
+Write `movies` in the name field and click on "Create Index":
+
+![A modal window with one mandatory field: "Index name"](https://raw.githubusercontent.com/meilisearch/documentation/cloud-quick-start/assets/images/cloud-getting-started/7-create-index.png)
+
+The final step in creating an index is to add data to it. Choose "File upload":
+
+![Another modal window with three options. A red arrow points at the chosen option, "File upload"](https://raw.githubusercontent.com/meilisearch/documentation/cloud-quick-start/assets/images/cloud-getting-started/8-file-upload.png)
+
+Meilisearch Cloud will ask you for your dataset. To follow along with this tutorial, use this <ButtonLink as="a" id="downloadMovie" href="/movies.json" download="movies.json">list of movies</ButtonLink>. Download the file to your computer, drag and drop it into the specified area, then click on "Import documents":
+
+![Another modal window with a large drag-and-drop area. It indicates a file named "movies.json" will be uploaded](https://raw.githubusercontent.com/meilisearch/documentation/cloud-quick-start/assets/images/cloud-getting-started/9-data-import.png)
+
+Meilisearch Cloud will index your documents. This may take a moment. Click on "See index list" and wait. Once it is done, click on "Settings" to visit the index overview:
```

Comment:
Should we add a screen of the index overview? Do you plan to go over index settings in another guide?

Reply:
We will add more in-depth guides as time goes on. Talking about index settings here would make the guide more complex than it should be, in my opinion.